packages/versioned-components/src/SquigglePlaygroundVersionPickerDropdown.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+import { FC, PropsWithChildren } from "react";
+
+import {
+  CodeBracketIcon,
+  Dropdown,
+  DropdownMenu,
+  DropdownMenuActionItem,
+  DropdownMenuHeader,
+  DropdownMenuLinkItem,
+  DropdownMenuModalActionItem,
+  DropdownMenuSeparator,
+  HelpIcon,
+  Modal,
+  WrenchIcon,
+} from "@quri/ui";
+
+import {
+  checkSquiggleVersion,
+  SquiggleVersion,
+  squiggleVersions,
+} from "./versions.js";
+
+export function versionTitle(version: SquiggleVersion) {
+  return version === "dev" ? "next" : `v${version}`;
+}
+
+export function uncheckedVersionTitle(version: string) {
+  const versionIsValid = checkSquiggleVersion(version);
+  if (versionIsValid) {
+    return versionTitle(version);
+  } else {
+    return `${version} (unknown)`;
+  }
+}
+
+export function versionIcon(version: string) {
+  return version === "dev" ? WrenchIcon : CodeBracketIcon;
+}
+
+const CHANGELOG_URL = "https://www.squiggle-language.com/docs/Changelog";
+
+export const SquigglePlaygroundVersionPickerDropdown: FC<
```

Comment:
I changed this from  ``SquigglePlaygroundVersionPicker``, so that it could be called with custom children. The playground/website want to call it with CSS based on the Playground, which gets messy when it's attempted to be coupled with this component directly.

Reply:
I'm not sure I understand the problem, and I see issues with this new approach:
- you had to duplicate more code in website and hub
- I see that you import `PlaygroundToolbarItem` from `@quri/squiggle-components` in the website and in the hub, so it's unversioned, and not future-proof: when the playground design (e.g. toolbar background color or height) changes again, both website and hub would have to be updated somehow, and it might get messy

It's possible to export `PlaygroundToolbarItem` through versioned-components, but that's a headache.

> The playground/website want to call it with CSS based on the Playground

If this is some technical issue that you encountered, maybe with Tailwind or with CSS inherited, maybe we can find another way to fix it?